EIPS/eip-1010.md

```diff
@@ -66,4 +66,4 @@ to ensure that backwards compatibility is not a concern.
 
 ## Copyright
 Copyright and related rights waived via
-[CC0](https://creativecommons.org/publicdomain/zero/1.0/).
+[CC0](../LICENSE.md).
```

Comment:
```suggestion
 Copyright and related rights waived via [CC0](../LICENSE.md).
```
Would be nice if we synchronized all of these since we are touching them all.

Reply:
Assuming that this was a desync error. Please unresolve if I've done something stupid.